structures.c

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>

```

Comment:
replace this to .h file

Reply:
#include ".....h"
Also it is better to split each structure in separate file, U can do it later...